lighthouse-core/config/default.json

```diff
@@ -297,7 +299,8 @@
       "audits": {
         "unused-css-rules": {},
         "uses-optimized-images": {},
-        "uses-responsive-images": {}
+        "uses-responsive-images": {},
+        "uses-request-compression": {}
```

Comment:
This needs a rebase since the objects here are required now for the auto-expanding of categories to work (this also got moved to perf recently I believe)